### PR TITLE
Add table for list of backend supported

### PR DESCRIPTION
**Pull request by anirudhbagri**
#2010

### CODE REVIEW

**Comment by almarklein** [comment]
> but I also think most of our examples use purely lowercase.

I'd say list them lowercase here as well.

Maybe the column name "Package" should be "Backend"?

**Comment by djhoese** [comment]
So @anirudhbagri are you going to work on this any more and address the comments or now that hacktoberfest is done are you done?

**Comment by anirudhbagri** [comment]
Sure, I will work on it. 
I thought there are some discussion pending among the owners, i was waiting :)

**Comment by djhoese** [comment]
> I'd say list them lowercase here as well.
>
> Maybe the column name "Package" should be "Backend"?

@anirudhbagri I think just this change ^ and this will be good to go. So make the left column say "Backend" and make them all lowercase. Any further discussion from vispy maintainers can happen after this is merged and if anyone complains. Thanks for your help adding this.

**Comment by anirudhbagri** [comment]
@djhoese, I have made the changes. Can you take one more look at PR.

**Review by djhoese — APPROVED** [review]
Looks good. I'll take a screenshot of the website generated by the CI and post it here. If it looks good then we can merge. Thanks @anirudhbagri!

**Comment by djhoese** [comment]
image

Looks pretty good. I suppose we should probably list some dependencies for sdl2, wx, and pyglet, right? Otherwise users will get the idea that they don't need to install anything extra for those. Is it enough to just repeat the name of the backend for those for their dependencies? @almarklein any idea if other libraries/packages are needed for these backends?

**Comment by almarklein** [comment]
wx -> wxPython
pyglet -> pyglet
sdl2 -> pysdl2, and on Win/MacOS pysdl2-dll is recommended.

File: doc/installation.rst
Comment (djhoese) [inline doc/installation.rst:53]:
```suggestion
   * - SDL2
     - Stable
     - pysdl2 (pysdl2-dll recommend on MacOS/Windows)
   * - wx
     - Stable
     - wxPython
   * - pyglet
     - Stable
     - pyglet
```

**Comment by djhoese** [comment]
image

Looks good enough to me. Thanks again @anirudhbagri!

**Pull request merged**
